src/olympia/files/decorators.py

```diff
@@ -54,7 +54,7 @@ def last_modified(request, obj, key=None, **kw):
 
 
 def etag(request, obj, key=None, **kw):
-    return _get_value(obj, key, 'md5')
+    return _get_value(obj, key, 'sha256')
```

Comment:
On top of that, we probably should be using a weak etag for file browsing, because we only hash the actual file content, not anything else.